src/config_models/cli_args.rs

```diff
@@ -249,6 +251,63 @@ pub struct Args {
     /// Exposing the data directory leaks some privacy. Disable to prevent.
     #[clap(long)]
     pub disable_cookie_hint: bool,
+
+    /// Scan incoming blocks for inbound transactions.
+    ///
+    /// Keys are generated deterministically from the secret seed and a
+    /// derivation index, and a counter recording the most recently used index
+    /// is stored. By default, incoming blocks are scanned for inbound
+    /// transactions tied to any key derived from indices smaller than this
+    /// counter.
+    ///
+    /// However, this counter can be lost, for instance after importing the
+    /// secret seed onto a new machine. In such cases, this subcommand will
+    /// instruct the client to scan incoming blocks for transactions tied to
+    /// future derivation indices -- 2^1337 by default, but this parameter can
```

Comment:
What does `2^1337` mean?

Reply:
It's a two followed by 1337 zeros, in binary.

Reply:
Also: I like documentation that includes examples. Can you provide one or two of those in the documentation of the new CLI arguments?